README.md

```diff
@@ -69,6 +70,10 @@ This playbook uses [Kitchen](https://kitchen.ci/) for CI and local testing.
 
 ### Running the tests
 
+* Ensure you have checked out this repository to `elaticsearch`, not `ansible-elasticsearch`.
+* If you don't have a Gold or Platinum license to test with you can run the trial versions of the `xpack-upgrade` and `issue-test` suites by appending `-trial` to the `PATTERN` variable.
+* You may need to explicity specify `VERSION=7.x` if some suites are failing.
```

Comment:
Yeah I assumed that would default to 7.x, but for some suites if it's not explicitly defined in the command it fails. This doesn't show up in Jenkins because it always specifies VERSION=x

E.g. `make verify PATTERN=oss-upgrade-centos-7`

```shell
       TASK [elasticsearch : set_fact] ************************************************
       Thursday 31 October 2019  09:45:14 +0000 (0:00:08.938)       0:00:09.084 ******
       fatal: [localhost]: FAILED! => {"msg": "The task includes an option with an undefined variable. The error was: 'es_major_version' is undefined\n\nThe error appears to be in '/tmp/kitchen/elasticsearch/tasks/main.yml': line 3, column 3, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n\n- set_fact: \"es_major_version={{ es_version.split('.')[0] }}.x\"\n  ^ here\nWe could be wrong, but this one looks like it might be an issue with\nmissing quotes. Always quote template expression brackets when they\nstart a value. For instance:\n\n    with_items:\n      - {{ foo }}\n\nShould be written as:\n\n    with_items:\n      - \"{{ foo }}\"\n"}

       PLAY RECAP *********************************************************************
       localhost                  : ok=1    changed=0    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0

       Thursday 31 October 2019  09:45:14 +0000 (0:00:00.025)       0:00:09.110 ******
       ===============================================================================
       Gathering Facts --------------------------------------------------------- 8.94s
       elasticsearch : set_fact ------------------------------------------------ 0.03s
>>>>>> ------Exception-------
>>>>>> Class: Kitchen::ActionFailed
>>>>>> Message: 1 actions failed.
>>>>>>     Converge failed on instance <oss-upgrade-centos-7>.  Please see .kitchen/logs/oss-upgrade-centos-7.log for more details
>>>>>> ----------------------
>>>>>> Please see .kitchen/logs/kitchen.log for more details
>>>>>> Also try running `kitchen diagnose --all` for configuration
```

Reply:
OK it seems that `VERSION := 7.x` in the `Makefile` isn't forwarded to kitchen as environment variable, however if you override that variable with `make VERSION=7.x`, it is forwarded to kitchen.

I'll work on a fix for that.